<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Bunny---HP9919" failures="1" tests="3" name="Test" time="38.301" errors="0" timestamp="2024-07-31T16:06:20 IST">
  <testcase classname="Task.ContactsWithData" name="createContact" time="7.556"/>
  <testcase classname="Task.OrganizationWithIndustry" name="createOrganizationWithIndustry" time="8.585"/>
  <testcase classname="Task.Organazation" name="createOrganization" time="7.289">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Task.Organazation.createOrganization(Organazation.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- createOrganization -->
</testsuite> <!-- Test -->
